docs/lint_command.md

```diff
@@ -0,0 +1,51 @@
+### Lint
+
+Run lintings (flake8, mypy, pylint, bandit) and pytest.
+pylint and pytest will run within all the docker images of an integration/script. Meant to be used with integrations/scripts that use the folder (package) structure. Will lookup up what docker image to use and will setup the dev dependencies and file in the target folder.
```

Comment:
Add a break line before this one. 
+
The appropriate docker image for the integration/script will be used to execute the pytest and pylint checks.

Reply:
done